content-based-message-routing/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "ballerinademo"
```

Comment:
This was for backends, not for tutorials.

Reply:
Updated